File: LTNC/bankaccount.java
class BankAccount{
    private double balance;
    private String password;

    public BankAccount(String pwd, double initBalance){
        this.balance = initBalance;
        this.password = pwd;
    }
    public double getBalance(String pwd){
        if(this.password.equals(pwd)){
            return this.balance;
        } else {
            return -1;
        }
    }
    public void deposit(String pwd, double amount){
        if(this.password.equals(pwd) && amount > 0){
            this.balance += amount;
        }
    }
    public void withdraw(String pwd, double amount){
        if(this.password.equals(pwd) && amount > 0 && amount <= this.balance){
            this.balance -= amount;
        }
    }
}

File: LTNC/bankaccount.py
class BankAccount:
    def __init__(self, initBalance: float = 0, password: str = "") -> None:
        self.balance: float = initBalance
        self.password: str = password
    
    def getBalance(self,pwd:str) -> float:
        if( pwd == self.password):
            return self.balance
        return -1.0
    
    def deposit(self, amount:float) -> None:
        self.balance += amount
    
    def withdraw(self, amount:float, pwd:str) -> None: 
        if( pwd == self.password and amount <= self.balance):
            self.balance -= amount